### DISCUSSION TITLE
How to use self annotated object detection boxes for key point recognition in the inference stage.

### DISCUSSION THREAD

**Discussion by SCsleepy** [open-mmlab/mmpose#2809] [Q&A] [answered]
Hi, I am using rtmpose to identify key points about animals. My question is how to use my annotated object detection box for key point recognition during the inference stage, rather than providing a trained object detection model?

Thank you.

**Comment by Ben-Louis** [comment, marked as answer]
This function is not yet supported. You could modify the code snippet for bounding box detection in https://github.com/open-mmlab/mmpose/blob/main/demo/topdown_demo_with_mmdet.py#L29-L44 to use annotated box.

**Reply by SCsleepy** [reply]
Thank you very much for your reply, it has been very helpful to me.
